test/e2e/utils/record-helpers.js

```diff
@@ -237,6 +237,24 @@ exports.testPresentation = function (tableParams) {
             expect('Encountered an error').toBe('Please check the log', 'Inside catch block');
         })
     });
+	
+	it("visible column related table with inline inbound fk should display 'None' in markdown disply mode if no data was found.",function(){
+        chaisePage.waitForElement(element(by.id('entity-booking'))).then(function(){
+            return browser.executeScript("return $('#entity-booking .btn-group .delete-action-button')");
+        }).then( function(deleteButtons){
+			// browser.pause();
+			for(var i = 0; i < deleteButtons.length; i++){
```

Comment:
weird indentation on this line and the next 3 lines. Maybe it's tab 4 spacing as opposed to tab 2 spacing?